static/js/stream_create.js

```diff
@@ -169,6 +169,13 @@ function create_stream() {
 
```

Comment:
generate_secrets.py.isorted should not be here.

Reply:
I do not understand this comment. Maybe this is a typo.

Reply:
Oh, I think I removed it in my push back yesterday, but you had added an extra file by that name  to the codebase.